aten/src/ATen/native/nested/cuda/NestedTensorTransformerFunctions.cu

```diff
@@ -1,3 +1,4 @@
+#include <cuda_fp16.h>
```

Comment:
I didn't review this file in much detail, I'm just assuming it's not significantly changed from fbgemm :)

Maybe can we add a code ptr to the fbgemm github repo so people can get more context?

Reply:
ah yes good call! will add an fbgemm pointer